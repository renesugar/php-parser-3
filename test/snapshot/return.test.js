const parser = require('../main');

describe("return", function() {
  it("simple", function() {
    expect(parser.parseEval('return "string";')).toMatchSnapshot();
  });
  it("no expression", function() {
    expect(parser.parseEval('return;')).toMatchSnapshot();
  });
});
