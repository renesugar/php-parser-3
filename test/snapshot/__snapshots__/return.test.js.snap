// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`return no expression 1`] = `
Program {
  "children": Array [
    Return {
      "expr": null,
      "kind": "return",
    },
  ],
  "errors": Array [],
  "kind": "program",
}
`;

exports[`return simple 1`] = `
Program {
  "children": Array [
    Return {
      "expr": String {
        "isDoubleQuote": true,
        "kind": "string",
        "raw": "\\"string\\"",
        "unicode": false,
        "value": "string",
      },
      "kind": "return",
    },
  ],
  "errors": Array [],
  "kind": "program",
}
`;
